README.md

```diff
@@ -1,31 +1,109 @@
-Docker configurations for Odoo Production environments
+DOCKER CONFIGURATIONS FOR ODOO PRODUCTION ENVIRONMENTS
 ======================================================
 
+## Table of Contents
+- [Installation](#installation)
+    - [Pre-installation](#pre-installation)
+    - [Docker installation](#docker-installation)
+- [Start-up](#start-up)
+    - [docker-compose](#docker-compose)
+    - [Step by step](#step-by-step)
+- [Useful commands](#useful-commands)
+    - [Docker Compose](#docker-compose-commands)
+    - [Docker Containers](#docker-container)
+    - [Docker Images](#docker-images)
+    - [Docker Networks](#docker-networks)
+    - [Docker Volumes](#docker-volumes)
+
+Installation
+=
+- ## Pre-installation
+    - Update the OS
+
+    <pre>
+    sudo apt update ()
```

Comment:
What is `()` ?

Reply:
This () is a typo.
I gonna solve it in the next commit.